### DISCUSSION TITLE
'connect ETIMEDOU'T when attempting to load data from MySQL

### DISCUSSION THREAD

**Discussion by AndyGUK** [Budibase/budibase#4937] [Help]
I've had a MySQL database connected to budibase for months and all of a sudden I'm getting 'connect ETIMEDOUT' whenever I try to access the data via my app.

I'm currently using budibase cloud.

The budibase IP is whitelisted. The credentials have been checked, updated and checked again. I can't figure out what's causing the issue. I feel like this has all started from around the time that I updated to the latest version, but this might not be true as I've backdated and I continue to get the same issue.

Has anyone experienced this before or know what the problem might be?

**Comment by mjashanks** [comment]
Hey Andy - I can verify this - looking into it now.
